docs/languages/rust.rst

```diff
@@ -14,9 +14,12 @@ The BridgeStan Rust client is available on `crates.io <https://crates.io/crates/
 
     cargo add bridgestan
 
-The first time you compile a model, the BridgeStan source code will be downloaded to `~/.bridgestan`. If you prefer to use a source distribution of BridgeStan, you can pass its path as the `bs_path` argument to `compile_model`.
+A copy of the BridgeStan C++ sources is needed to compile models. This can be downloaded to
```

Comment:
"can be" -> "is"

Reply:
Unlike Julia, etc, in Rust we do make the user actually call a separate function to download, so I think "can be" is more accurate